entrypoint.sh

```diff
@@ -56,6 +56,11 @@ if ! flyctl status --app "$app"; then
 
   flyctl deploy --app "$app" --region "$region" --image "$image" --region "$region" --strategy immediate --ha=false
 elif [ "$INPUT_UPDATE" != "false" ]; then
+  # PR opened - add postgres db app
+  if [ "$EVENT_TYPE" = "opened" ]; then
+    flyctl postgres create --name "$db_app" --org "$org" --region "$region" --initial-cluster-size 1 --volume-size 1 --vm-size shared-cpu-1x --autostart || true
```

Comment:
@LikHangg What does the `|| true` do at the end?

Reply:
It's going to continue on the next line even if the create of the db has failed. Same with the attaching db app.

Reply:
Any reasons why we want it to continue the next line if the pg create fail? Or attaching

Reply:
I was thinking even if it's failed to create, we still can manually create the db app and attach it. But it seems to be redundant. I'll remove it.